lib/plugins/aws/deploy/compile/events/schedule/index.js

```diff
@@ -61,6 +83,8 @@ class AwsCompileScheduledEvents {
                   "ScheduleExpression": "${ScheduleExpression}",
                   "State": "${State}",
                   "Targets": [{
+                    ${Input ? `"Input": "${Input}",` : ``}
+                    ${InputPath ? `"InputPath": "${InputPath}",` : ``}
```

Comment:
@kbariotis, I sent you a pull request on your fork so we can quickly integrate this change...